src/hotspot/cpu/x86/stubGenerator_x86_64.cpp

```diff
@@ -4179,6 +4179,28 @@ void StubGenerator::generate_compiler_stubs() {
       = CAST_FROM_FN_PTR(address, SharedRuntime::montgomery_square);
   }
 
+  // Load x86_64_sort library on supported hardware to enable avx512 sort and partition intrinsics
+    if (UseAVX > 2 && VM_Version::supports_avx512dq()) {
```

Comment:
Indention (spacing) is wrong for lines 4183-4190 after you moved check.

Reply:
Fixed the indentation. Please see the latest commit that was pushed.